code/controllers/subsystem/materials.dm

```diff
@@ -47,6 +50,8 @@ SUBSYSTEM_DEF(materials)
 			continue // Do not initialize at mapload
 		InitializeMaterial(list(mat_type))
 
+	dimensional_themes = init_subtypes_w_path_keys(/datum/dimension_theme)
```

Comment:
Small thing about making them singletons, while most of the themes don't hold state, `/datum/dimension_theme/fancy` does. It randomly selects 1 carpet type to use as a floor.

This means the fancy theme will always pick the same carpet on a per-round basis. If you're ok with that then it's whatever, just worth pointing out.

Reply:
Oh, thank you. I'll make sure to fix that.